R/derive_var_ady.R

```diff
@@ -38,22 +40,15 @@
 #'
 #' @export
 #'
-#' @examples
```

Comment:
Should we put this in the deprecation strategy to remove examples from deprecated functions?

Reply:
I put it there.

Reply:
Oh lol I was looking at deprecation strategy for 0.6.3 website and not the updated 0.7.0 website.  Thanks!